### ISSUE TITLE
Invalid use of void expression when compiling

### ISSUE THREAD

**Issue by aviezab** [lwfinger/realtek_bluetooth#3]
```
realtek_bluetooth/btusb.c:1744:19: error: invalid use of void expression
 1744 |  *skb_put(skb, 1) = 0x00;
```

GCC = 9.3.0
CLang = 10.0.1
OS Arch Linux with Kernel 5.5.9

Please help us to fix this.

**Comment by lwfinger** [comment]
I fixed this repo; however, with kernel 5.5.9, you should be using the driver in the kernel. This one is copied from the kernel for use with really old kernels.

**Comment by aviezab** [comment]
Thank You @lwfinger , I don't know why the Kernel detected it well, also rfkill. But, bluez and other bluetooth client stuffs don't detect the bluetooth well, so I am trying to use this.
